src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -90,13 +90,25 @@ public function onRespond(ResponseEvent $event) {
       }
 
       $tags_string = implode(' ', $tags);
-      if (25000 < strlen($tags_string)) {
-        $tags_string = substr($tags_string, 0, 25000);
-        // The string might have cut of in the middle of a tag.
-        // So now find the the last occurence of a space and cut to that length.
+      // value also defined in config install and schema files
+      $limit = 25000;
+
+      // Allow to lower the surrogate key header limit on non-Pantheon Env
+      if (!isset($_ENV['PANTHEON_INFRASTRUCTURE_ENVIRONMENT']) ||
+        $_ENV['PANTHEON_INFRASTRUCTURE_ENVIRONMENT'] === 'lando') {
```

Comment:
```suggestion
      if (!isset($_ENV['PANTHEON_INFRASTRUCTURE_ENVIRONMENT'])) {
```
My previous suggestion re: lando only mattered for PANTHEON_ENVIRONMENT. Greg's suggestion of `PANTHEON_INFRASTRUCTURE_ENVIRONMENT` is better, as Lando does not use it.

```
lando exec appserver -- printenv | grep PANTHEON
PANTHEON_INDEX_SCHEME=http
PANTHEON_INDEX_CORE=/lando
PANTHEON_INDEX_HOST=index
PANTHEON_SITE=<...>
PANTHEON_INDEX_PATH=/
PANTHEON_INDEX_PORT=449
PANTHEON_SITE_NAME=<...>
PANTHEON_INDEX_SCHEMA=solr/#/lando/schema
PANTHEON_ENVIRONMENT=lando
```